preditor/gui/console.py

```diff
@@ -213,6 +253,11 @@ def errorHyperlink(self):
             return
 
         if modulePath:
+            # Wrap filepaths in quotes in case they contain spaces
+            pattern = r"(?<!\")({\w+Path})(?!\")"
+            repl = r'"\g<1>"'
+            cmdTempl = re.sub(pattern, repl, cmdTempl)
```

Comment:
Why can't the user(or more accurately our default) for `cmdTempl` just correctly add double quotes? I think the only correct way to handle auto-quoting subprocess calls is to pass a list of strings, and try to avoid passing a string to subprocess unless I have no other choice.

Reply:
On windows I changed my template to `"{exePath}" "{modulePath}":{lineNum}` and it seemed to fix the issue. This gives me control over the command being run.

Reply:
It's been a while since I wrote this, but I think I chose this way because anybody who has already run PrEditor would not get this default, unless they wipe their prefs, or go in and wipe this particular pref.

Reply:
I was thinking of a way to force reload the default once, but what if user already set a different command?

Reply:
Maybe, keep the regex bit, and notice if it's different than what the cmdTempl was originally, and in that case, notify user to updated, rather than being a hidden change every time.

Reply:
Looks like you have added a checker that makes sense on windows, and I don't think it will cause issues for bash file paths that can use double quotes for paths.